recipe/meta.yaml

```diff
@@ -32,10 +32,11 @@ test:
     - pip check
     - pytest -vv --cov=jupyter_server_terminals --cov-report=term-missing:skip-covered --no-cov-on-fail --cov-fail-under=80
   requires:
-    - jupyter_server >=2.0.0rc5
+    - jupyter_server >=2.0.0rc8
     - pip
     - pytest >=7.0
     - pytest-cov
+    - pytest-jupyter >=0.5.3
```

Comment:
woudl be good to get https://github.com/conda-forge/pytest-jupyter-feedstock/pull/1

Reply:
[3bdbcbd](https://github.com/conda-forge/jupyter_server_terminals-feedstock/pull/3/commits/3bdbcbd426f4c7bc680ec571e71cb8cf9f2804be)